docs/ref/forms/fields.txt

```diff
@@ -1260,13 +1260,47 @@ method::
 
     .. attribute:: queryset
 
-        Same as :class:`ModelChoiceField.queryset`.
+        A ``QuerySet`` of model objects from which the choices for the field
+        are derived and which is used to validate the user's selection. It's
+        evaluated when the form is rendered.
 
     Takes one optional argument:
 
     .. attribute:: to_field_name
 
-        Same as :class:`ModelChoiceField.to_field_name`.
+        This optional argument is used to specify the field to use as the value
+        of the choices in the field's widget. Be sure it's a unique field for
+        the model, otherwise the selected value could match more than one
+        object. By default it is set to ``None``, in which case the primary key
+        of each object will be used. For example::
+
+            # No custom to_field_name
+            field1 = forms.ModelMultipleChoiceField(queryset=...)
+
+        would yield:
+
+        .. code-block:: html
+
+            <select id="id_field1" name="field1" multiple>
+            <option value="obj1.pk">Object1</option>
+            <option value="obj2.pk">Object2</option>
+            ...
```

Comment:
This should be an ellipse, not 3 dots, right? Maybe we should scratch the line all together, since it not really HTML. As an alternative we would add a `<!-- -->` comment wrapper

Reply:
Using three dots in HTML examples (and other code blocks) is an existing pattern found throughout the docs. Here are the ones I quickly found on `master`:

```
docs/ref/forms/fields.txt
1202         .. code-block:: html
1217         .. code-block:: html

docs/ref/forms/api.txt
951      .. code-block:: html

docs/ref/forms/widgets.txt
692      .. code-block:: html
786      .. code-block:: html
```

This new doc is following that pattern. In fact, it used the `docs/ref/forms/fields.txt` as a starting point. I agree three dots here is not valid HTML, but I think its intent as a placeholder is clear to the reader.

Reply:
Currently, it is a mix, which I'd like to avoid. In `docs/topics/serialization.txt` and other we use the correct comment version.
Since you are adding documentation here, I believe it's fair to asses which way to do it going forward. I personally believe the commented way to better. It being valid HTML after all.

Ellipses vs. dots is similar. We underwent a lot of effort to fix it in the code. I believe we don't need to stop with the documentation. An ellipsis is a Unicode character, why not use it?
We also do use ellipsis in all sentences in the documentation. Only in code we have dots left.